release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1727,10 +1732,55 @@ If these values are not defined in a failure domain configuration, for instance
 * Previously, on on the *Observe -> Alerting -> Silences* tab, the `creator` field was not autopopulated and was not designated as mandatory. This issue happened when the API made the field empty from {product-title} 4.15 and onwards. With this update, the field is marked as mandatory and populated with the current user for correct validation. (link:https://issues.redhat.com/browse/OCPBUGS-35048[*OCPBUGS-35048*])
 
 
+[discrete]
+[id="ocp-release-note-oc-mirror-bug-fixes_{context}"]
+==== oc-mirror
+
+* Previously, when using `oc-mirror --v2 delete --generate`command, the contents of the working-dir/cluster-resources folder were cleared. With this fix, the working-dir/cluster-resources folder is not cleaned when the delete feature is used. (link:https://issues.redhat.com/browse/OCPBUGS-48430[*OCPBUGS-48430*])
+
+* Previously, release images were signed using a SHA-1 key. On {op-system-base} 9 FIPS STIG-compliant machines, verification of release signatures using the old SHA-1 key failed due to security restrictions on weak keys. With this release, release images are signed using a new SHA-256 trusted key so that the release signatures no longer fail. (link:https://issues.redhat.com/browse/OCPBUGS-48314[*OCPBUGS-48314*])
+
+* Previously, when running oc-mirror v2 Technology Preview in mirror-to-disk mode within an disconnected environment, the tool attempted to download `graph.tar.gz` from `api.openshift.com`, causing the mirroring process to fail.
```

Comment:
```suggestion
* Previously, when running oc-mirror v2 Technology Preview in mirror-to-disk mode within a disconnected environment, the tool attempted to download the `graph.tar.gz` file from the `api.openshift.com` website, causing the mirroring process to fail. 
```
solution and bug link are missing

Reply:
Some of these bugs read like we have a TP version that is still supported and that 4.18 is an updated GA. Is 4.18 the GA of v2? If so, I think removing the TP mentions completely is more clear. Otherwise, it sounds like I might still be using the TP version (and that's OK). I think we want folks to update to the GA release, yes?

Reply:
Confirmed with our DPM to comment these out and have the writer review them next week for late inclusion. Nice catch!